src/Strategy/ExplodeStrategy.php

```diff
@@ -27,29 +37,17 @@
      * @param string   $delimiter    String that the values will be split upon
      * @param int|null $explodeLimit Explode limit
      */
-    public function __construct($delimiter = ',', $explodeLimit = null)
+    public function __construct(string $delimiter = ',', ?int $explodeLimit = null)
     {
         $this->setValueDelimiter($delimiter);
-
-        $this->explodeLimit = ($explodeLimit === null) ? null : (int) $explodeLimit;
+        $this->explodeLimit = ($explodeLimit === null) ? null : $explodeLimit;
```

Comment:
This can be replaced with simple assignement `$this->explodeLimit = $explodeLimit;`

Reply:
Good catch!